pkg/api/targets.go

```diff
@@ -67,3 +69,34 @@ func (api *API) GetTargetMTTR(c echo.Context) error {
 
 	return response(c, mttr, "mttr")
 }
+
+// DeleteTargetTag deletes the association between a target and a tag.
+func (api *API) DeleteTargetTag(c echo.Context) error {
+	targetID := c.Param("target_id")
+	tag := c.Param("tag")
+
+	err := api.storage.DeleteTargetTag(targetID, tag)
+	if err != nil {
+		if errors.Is(err, sql.ErrNoRows) {
```

Comment:
Actually, this should never happens, since the authorisation routine will prevent the user to reach this point (the tag wont exists for this asset)

Reply:
But I think we should not assume that right? Maybe at some point this method is executed from a different flow.
What do you think?

Reply:
Fair enough, just wanted to confirm.